aten/src/ATen/miopen/Descriptors.cpp

```diff
@@ -90,17 +90,17 @@ std::ostream& operator<<(std::ostream & out, const TensorDescriptor& d) {
 
 void TensorDescriptor::print() { std::cout << *this; }
 
-void FilterDescriptor::set(const at::Tensor &t, int64_t pad) {
+void FilterDescriptor::set(const at::Tensor &t, const at::MemoryFormat memory_format, int64_t pad) {
```

Comment:
I actually think passing memory format here explicitly is kind of suspect, but it is symmetric with cuDNN so I'll let it slide.

Reply:
Yes, I referred to the cuDNN implementation and tried to match it as much as possible.